samples/getting-started/azure-quantum/Getting started with Cirq and Honeywell on Azure Quantum.ipynb

```diff
@@ -0,0 +1,331 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8756da21",
+   "metadata": {},
+   "source": [
+    "# Getting started with Cirq and Honeywell on Azure Quantum\n",
+    "\n",
+    "This notebooks shows how to send a basic quantum circuit to a Honeywell target via Azure Quantum."
```

Comment:
To outside users who already have the context of a "cloud service" I think "Azure Quantum" is more clear.

Reply:
Fair, but we've also used that term very explicitly to include not just the cloud service, but also the client SDK and local-only workflows. For instance, the entire docs site is under docs.microsoft.com/azure/quantum, including components unrelated to services.

Reply:
That's true but what I meant with context is that for users who are comparing cloud services I imagine they won't append the word "service" behind everything, i.e. they will compare <company name> not <company name> service. This might be something for @bromeg to chime in on.

Reply:
@cgranade , I just discussed with @bromeg  and seeing how we are using the term in other parts of the docs she thinks it will be OK to use here and wouldn't create any confusion for the users.